frontend/src/components/pages/RegistrantExperience/RegistrationSteps/AdditionalInfo/index.tsx

```diff
@@ -1,26 +1,90 @@
-import React from "react";
-import { Box, Checkbox, Text } from "@chakra-ui/react";
+import { Box, Divider, Text, VStack } from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+import { RegistrantExperienceCamper } from "../../../../../types/CamperTypes";
+import { FormQuestion } from "../../../../../types/CampsTypes";
+import { useAdditionalInfoDispatcher } from "./additionalInfoReducer";
+import CamperQuestionsCard from "./QuestionCards/CamperQuestionsCard";
+import EarlyDropOffLatePickupCard from "./QuestionCards/EarlyDropOffLatePickupCard";
 
 type AdditionalInfoProps = {
-  isChecked: boolean;
-  toggleChecked: () => void;
+  nextBtnRef: React.RefObject<HTMLButtonElement>;
+  campers: RegistrantExperienceCamper[];
+  setCampers: React.Dispatch<
+    React.SetStateAction<RegistrantExperienceCamper[]>
+  >;
+  campName: string;
+  formQuestions: FormQuestion[];
+  hasEarlyDropOffLatePickup: boolean;
+  requireEarlyDropOffLatePickup: boolean | null;
+  setRequireEarlyDropOffLatePickup: React.Dispatch<
+    React.SetStateAction<boolean | null>
+  >;
 };
 
 const AdditionalInfo = ({
-  isChecked,
-  toggleChecked,
+  nextBtnRef,
+  campers,
+  setCampers,
+  campName,
+  formQuestions,
+  hasEarlyDropOffLatePickup,
+  requireEarlyDropOffLatePickup,
+  setRequireEarlyDropOffLatePickup,
 }: AdditionalInfoProps): React.ReactElement => {
+  const dispatchAdditionalInfoAction = useAdditionalInfoDispatcher(setCampers);
+
+  const [submitClicked, setSubmitClicked] = useState(false);
```

Comment:
can we call this `nextClicked` since submit indicates that this is the final step